src/app/consumption-estimation/consumption-estimation.component.html

```diff
@@ -34,17 +33,14 @@ <h2 class="form-header">Service</h2>
                 <h2 class="form-header">Address</h2>
             
                 <div class="flex place-content-evenly ">
-                    <input type="text" formControlName="city"  placeholder="City" class="mr-1"/>        
-                    <input type="text" formControlName="postcode"  placeholder="Postcode"/>
+                    <mat-select formControlName="addressID" [(value)]="selectedAddressID">
+                        <mat-option  *ngFor="let address of fullAddresses; let i=index" [value]="addressesIDs[i]">{{address}}</mat-option>
+                      </mat-select>
```

Comment:
I dont like this cross array stuff.
Better off with let address of addresses and then doing address.id, then u dont need the index either